src/test/java/BaseTest.java

```diff
@@ -10,4 +21,30 @@ public static void chromeConfigs() {
         }
     }
 
-}
+    @BeforeClass
+    @Parameters({"BaseURL"})
+    public void launchBrowser(String BaseURL) {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("--disable-notifications");
+
+        driver = new ChromeDriver(chromeOptions);
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        url = BaseURL;
+        driver.get(url);
+    }
+    protected WebElement waitForElementToBeClickable(WebElement webElementLocator) {
+      return new WebDriverWait(driver, Duration.ofSeconds(10)).until(
+             ExpectedConditions.elementToBeClickable(webElementLocator));
+    }
+
+    protected WebElement waitForVisibilityOfElement(WebElement webElementLocator) {
+        return new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions
+               .visibilityOf(webElementLocator));
```

Comment:
**Needless Repetition!** You can remove these methods because they are in the `class BasePage`